### PR TITLE
try to split testsuites into two

### PR DESCRIPTION
**Pull request by skshetry**
Trying to see if we can improve CI runtime speed.

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -43,6 +43,9 @@ jobs:
       matrix:
         os: [ubuntu-20.04, windows-latest, macos-latest]
         pyv: ["3.8", "3.9", "3.10"]
+        pytest-filter: 
+        - "import or plot or live or experiment"
```
Comment (skshetry) [inline .github/workflows/tests.yaml:47]: Using `import or plot or live or experiment` to try to split Windows tests in half.

**Review by efiop — COMMENTED** [review]
So wha'ts slow in experiments and those tests that you are splitting?

This looks pretty ugly, so I'm wondering if there are any better alternatives.

**Comment by skshetry** [comment]
> So wha'ts slow in experiments and those tests that you are splitting?

Experiments are slow, yes, but the reason why I am splitting these are that Windows takes 22 minutes to run.
And it's not just experiments, there are many tests that are slow that are imports or filesystems or data management, etc. but they are random and unpredictable. I have doubt if `pytest-xdist` is to blame for them to be unpredictably slow.
I did raise an issue internally about a month ago about this.

> This looks pretty ugly, so I'm wondering if there are any better alternatives.

Splitting the test suites is the next logical step in these cases.

**Comment by skshetry** [comment]
I'm interested in moving forward with this if there are no strong reasons.
If we get any issues, I'm fine with reverting this back. The investigation on why it's slow is a separate discussion.

**Comment by efiop** [comment]
Alright, feel free to merge.

**Comment by skshetry** [comment]
At this point with these slow and flakey tests, I think we are fighting a losing battle. 😄 
So that's why I have been so protective on `dvc-objects`/`dvc-data` side regarding tests, using `MemFS`, writing good tests strategically, making tradeoffs between maintaining the test vs the value it brings, etc.

**Pull request merged**